persistence-testkit/src/main/scala/org/apache/pekko/persistence/testkit/PersistenceTestKitPlugin.scala

```diff
@@ -112,9 +112,12 @@ object PersistenceTestKitPlugin {
  * Persistence testkit plugin for snapshots.
  */
 @InternalApi
-class PersistenceTestKitSnapshotPlugin extends SnapshotStore {
+class PersistenceTestKitSnapshotPlugin(@unused cfg: Config, cfgPath: String) extends SnapshotStore {
```

Comment:
why do we need an unused parameter?

Reply:
It is because of this bit of code:
https://github.com/apache/pekko/blob/242145821f471c1929a32142caeeb55edfd3450f/persistence/src/main/scala/org/apache/pekko/persistence/Persistence.scala#L413-L425
Easiest solution seemed to work around this the same as it is done in  `PersistenceTestKitPlugin`.

Reply:
so if the param is used, can you removed the `@unused` annotation?

Reply:
It is used only through reflection, so compilation throws the following if `@unused` is removed:
```
parameter cfg in class PersistenceTestKitSnapshotPlugin is never used
[error] Applicable -Wconf / @nowarn filters for this fatal warning: msg=<part of the message>, cat=unused-params, site=org.apache.pekko.persistence.testkit.PersistenceTestKitSnapshotPlugin.cfg                                                                         
[error] class PersistenceTestKitSnapshotPlugin(cfg: Config, cfgPath: String) extends SnapshotStore {                                                                                                                                                                     
[error]                                        ^                                                                                                                                                                                                                         
[error] one error found
[error] (persistence-testkit / Compile / compileIncremental) Compilation failed
```

Reply:
fair enough - could you add a code comment explaining the `@unused`?